### ISSUE TITLE
Problem with missing audio - prevent warning clip

### ISSUE THREAD

**Issue by elemn** [yomidevs/local-audio-yomichan#26]
Hello,

I have the mp3 version of this library installed  locally, it works great! I've recently decided to enable autoplay for when I'm reading and looking up words, but I often have this issue where there is no availible audio, and I get the message "The audio for this clip is currently unavailable, it will be uploaded shortly, thank you for your patience.", this happens quite frequently, possibly because some of my dictionaries list certain less common word forms. 

Is it possible to just give a "click" sound or something else instead? Before I had this add on, I was using web sources and that's what would happen in such cases. It gets a bit grating hearing this all the time. I have no other sources enabled. Is there a modification I can do to the database?

**Comment by Mansive** [comment]
I believe this is an issue fixed by updating from Yomichan to Yomitan.

**Comment by elemn** [comment]
@Mansive Gotcha, thanks for the heads up ! I'll look into updating one of these days and report back.

**Comment by elemn** [comment]
Yes that indeed fixed it. Guess this is what finally got me to convert.

Thanks again!
